view/frontend/templates/catalog/product/view/type/grouped/product-protection-offer.phtml

```diff
@@ -28,7 +30,7 @@ if ($category = $block->getProduct()->getCategory()) {
                     {
                         "extendStoreUuid": "<?= $block->getData('viewModel')->getExtendStoreUuid() ?>",
                         "activeEnvironment": "<?= $block->getData('viewModel')->getActiveEnvironment() ?>",
-                        "selectedProductSku": "<?= $product->getSku(); ?>",
+                        "selectedProductSku": "<?= $encodedProductSku; ?>",
```

Comment:
i don't think we need the encoded uri here because these are parameters passed as config to the corresponding js function